cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -30,9 +30,12 @@ if(APPLE)
   set(configure_command "./Configure")
   set(configure_opts "darwin64-x86_64-cc")
```

Comment:
I think this branch should be updated too. Probably it make sense to add `CC=*` out of the `if` conditions (for both APPLE and non-APPLE).